recipes/matchms/meta.yaml

```diff
@@ -30,7 +30,9 @@ requirements:
     - rdkit >=2020.03.1
     - requests
     - scipy >=1.4.0
-    - pickydict >=0.3.0
+    - sparsestack >=0.4.1
+    - pickydict >=0.4.0
+    - pyyaml
```

Comment:
```suggestion
    - sparsestack >=0.4.1
    - pickydict >=0.4.0
    - pyyaml
    - tqdm
```

Reply:
Since I have no editing rights I added a new PR for this #38675 .